crt_portal/cts_forms/migrations/0051_auto_20200227_1408.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.9 on 2020-02-27 14:08
```

Comment:
Pretty sure this file was just renamed so it should just be deleted.

Reply:
Oops, good catch. Thanks! Deleted